### PR TITLE
Add renovate configuration to auth to jitpack

### PR DESCRIPTION
**Pull request by oscarnylander**
These credentials are encrypted and can only be used by renovate.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
   "extends": [
     "config:base"
+  ],
+  "hostRules": [
+    {
+      "hostType": "maven",
+      "matchHost": "https://jitpack.io",
+      "password": "",
+      "encrypted": {
+        "username": "wcFMA/xDdHCJBTolAQ//dmlKZCl7d/f5ogqq1JctWUVf9VFA8efOdQqQnQhVbE4vJUzG43//HsGWoF11VGkvVD4lYBVh/ALbf60cTtgwudECMoWDIKANXj0F2+oToDM9mCFuhn9qB0jRJS+axbfiSfoKvziIzM0Bbs34jx2rKHSv6FYOYweAg9z+LDRQmJiH5NfQQ+ykLAbgOKBShJBez+v05By4E21HKDwnurKgPUB4NtjRj4Oui0rnpbRexqtB3aRQJ2rTtiTyUnT02yBQBmtl7dyRge/uJU9i3eYmv/s4fQyVGTx5fdxFwGKoYE1UblA3TTIb/Ml4jb5v6uQQAZ2wppJPlwjXI/wbWG5JhhsaclTySLtwGZ9YYJoOaaM1y4YwcNmYxIddOBpTvEoq/13uFCwGnY+46LnUAJTIlnk2JwQCI+qq8nUU8o0HVYtkYd7FB3/X7JiWxLkOzc+TrUuaCjhVlTuCXWj5akXvjQl28NlWnL8d5iQVQ6WXUzLWzw/RSPsWvHRv+VxyjmjnMEZaHr9ZFwXXkAPTVy251xamGeU/Kv3OzqHMA9s/FkRXsPnbCGSXq7VMB4VeURLvQpcnpwDydMZEvQdJk7wH+zs7iLYziW5pjKHPenRuQjw6mLk9gGPphq+sYmKfigclRoOc+OJngR7AtGOgu3z31TtUjoAlKerS7nNShL5TyA7ScwFNmaoo/4xNIBhao7d9VyDuKH1DsYnnwLd2h/Q0zFryiPse06J+SSPrprDfqHIOhiRZNTlGOv4arhUhhcciOy2fPQyolNknamom4IJbiCOxOEmuxWZQbdlYDKil6YN88ZAv+RJny/xl9LHLEjoDKweIiTw"
```
[the remark this answers is not in the document]
Comment (oscarnylander) [reply renovate.json:11]: JSON files don't support comments, unfortunately 😬

[the remark this answers is not in the document]
Comment (hugokallstrom) [reply renovate.json:11]: Good idea!

**Review by hugokallstrom — COMMENTED** [review]
Nice, will hopefully make the odyssey release cycle a bit smoother 👍

File: renovate.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (StylianosGakis) [reply renovate.json:11]: Yeah hence I mentioned the README, I'd love to see it there at least 👀

[the remark this answers is not in the document]
Comment (oscarnylander) [reply renovate.json:11]: Do you have any suggestions on how to document it? I can't even think of a reasonable thing to write about it 😞

[the remark this answers is not in the document]
Comment (hugokallstrom) [reply renovate.json:11]: Suggestion: what the credentials are for, how they are encrypted, and where to find them

[the remark this answers is not in the document]
Comment (StylianosGakis) [reply renovate.json:11]: Hmm append in our existing top-level README.md something like:

```
## Renovate

`renovate.json` is a file which Renovate looks at in order to be able to suggest upgrading private artifacts to their latest version.
The `username` used in there is the result of passing our jitpack username (from 1Password) through Renovate's [website](insert link here) to get the hashed version of it.
```

Just so if we need to update it at some time in the future we can do so.

**Pull request merged**